src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -426,6 +426,38 @@
         az functionapp plan create -g MyResourceGroup -n MyPlan --number-of-workers 4 --sku S1
 """
 
+helps['functionapp plan update'] = """
+type: command
+short-summary: Update an App Service plan for an Azure Function.
+examples:
+  - name: Update an app service plan to S1 sku with four workers.
+    text: >
+        az functionapp plan create -g MyResourceGroup -n MyPlan --number-of-workers 4 --sku S1
+"""
+
+helps['functionapp plan delete'] = """
+type: command
+short-summary: Delete an App Service Plan.
+"""
+
+helps['functionapp plan list'] = """
+type: command
+short-summary: List App Service Plans.
+examples:
+  - name: List all free tier App Service plans.
+    text: >
+        az functionapp plan list --query "[?sku.tier=='Free']"
+"""
+
+helps['functionapp plan show'] = """
+type: command
+short-summary: Get the App Service Plans for a resource group or a set of resource groups.
+examples:
+  - name: Get the app service plans for a resource group or a set of resource groups. (autogenerated)
+    text: az appservice plan show --name MyAppServicePlan   --resource-group MyResourceGroup
```

Comment:
this should be `az functionapp plan show`

Reply:
Sorry, somehow missed this one